### ISSUE TITLE
Build instructions missing?

### ISSUE THREAD

**Issue by fps** [SubhamTyagi/Last-Launcher#132]
Hi,

I'm trying to build this app from source. Just loading it in the current Android Studio does not work (usually I would expect to be able to press "Run" and have it go to my emulator or the device, but Android Studio instead asks me to "Edit configurations...").

What steps are necessary to build the app from source?

Thanks for the awesome launcher by the way :)

**Comment by SubhamTyagi** [comment]
There no special build-instructions to build this app, follow the android studio guide.

**Comment by fps** [comment]
Hi again.

Android Studio is weird :) Anyways, I had to change the gradle and gradle plugin versions in the File/Project Structure settings to something more recent. Then suddenly the build worked fine. Previously I would always get "Gradle Sync: Sync failed: Unknown reason". Even the log didn't have any more helpful infos.

Possibly I made an error right at the start by just opening the project and not importing it. I will try from blank state and see if that makes a difference..
